### DISCUSSION TITLE
List of most popular packages

### DISCUSSION THREAD

**Discussion by EvanHahn** [npm/feedback#692] [Registry] [answered]
The npmjs.com homepage used to list the most popular packages, but no longer. Is there a way to get this data?

**Comment by monishcm** [comment, marked as answer]
I am able to see this in the npmjs homepage (after you login). Do you mean another issue? If so can you please elaborate with some screenshots?

![image](https://user-images.githubusercontent.com/23190045/172134924-8611ba78-2a1c-4d3f-a84d-6d1ce04ca6b6.png)

**Reply by EvanHahn** [reply]
That's it! I didn't realize I needed to log in.
